ddtrace/contrib/internal/langchain/patch.py

```diff
@@ -965,6 +965,170 @@ def traced_similarity_search(langchain, pin, func, instance, args, kwargs):
     return documents
 
 
+@with_traced_module
+def traced_base_tool_invoke(langchain, pin, func, instance, args, kwargs):
```

Comment:
I couldn't think of any metrics to emit from this function

Reply:
Are you talking about datadog metrics? If so we can ignore, that's out of scope for our team and this integration.

Reply:
Don't we use any metrics from those emitted from the langchain integration ?